vmdb/app/controllers/ems_infra_controller.rb

```diff
@@ -52,7 +52,18 @@ def scaling
           scale_parameters_formatted << {"name" => p.name, "value" => scale_parameters[p.name]}
         end
       end
-      if scale_parameters_formatted.length > 0
+
+      begin
+        # Check if stack is ready to be updated
+        update_ready = @stack.update_ready?(@stack.raw_status.first)
```

Comment:
Why is this code here?  the `scale_parameters_formatted` is above it but it doesn't seem to be used until after this `@stack.update_ready?` code.  Should this update_ready code come before the `scale_parameters_formatted` so the latter is all together?